caffe2/python/rnn_cell.py

```diff
@@ -622,7 +617,8 @@ def _apply(
                     layer_next_states,
                 )
                 if i > 0 and i in self.residual_output_layers:
-                    layer_input = model.net.Sum(
```

Comment:
This is failing with error:
  "indentation contains mixed spaces and tabs"

Reply:
Otherwise diff looks good

Reply:
Oops! Sorry about that.